src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -229,8 +229,16 @@ public static class ILoggerExtensions
             
             context = context ?? new Dictionary<string, object>();
 
-            logger.LogWarning(
-                DependencyFormat, dependencyType, dependencyData, targetName, duration, startTime.ToString(FormatSpecifiers.InvariantTimestampFormat), isSuccessful, context);
+            logger.LogWarning(DependencyFormat, new DependencyLogEntry(
+                dependencyType, 
+                dependencyName: null, 
```

Comment:
Why is dependencyName null ?

Reply:
I don't the exact reason, but we have chosen to not use this one in our custom dependency types. It's mostly used by the built-in ones.

Reply:
This should be added, see https://github.com/arcus-azure/arcus.observability/issues/165